kDriveFileProvider/FileProviderExtension.swift

```diff
@@ -401,16 +409,4 @@ final class FileProviderExtension: NSFileProviderExtension {
         let validationService = FileProviderValidationServiceSource(fileProviderExtension: self, itemIdentifier: itemIdentifier)!
         return [validationService]
     }
-
-    // MARK: - Async
-
-    /// Enqueue an async/await closure in the underlaying serial execution queue.
-    /// - Parameter task: A closure with async await code to be dispatched
-    func enqueue(_ task: @escaping () async throws -> Void) {
```

Comment:
This was extra complexity that is not helping, therefore I switched it for a `Task` everywhere.

Reply:
removed in 4.4.2-4